request.js

```diff
@@ -89,8 +89,15 @@ SuiteRequest.prototype = {
 
   _getLogParameters: function(options) {
     return _.pick(options, ['method', 'host', 'url']);
-  }
+  },
 
+  _getPayload: function(data) {
+    if (this._options.getHeader('content-type') !== 'application/json') {
```

Comment:
Made it more acceptable with indexOf, i dont think lowercasing is a real issue now, havent seen someone writing uppercase header values for this